package.json

```diff
@@ -67,6 +81,8 @@
     "sass": "^1.26.11",
```

Comment:
Update so we get deprecation messages when we build

Reply:
Current @^1.34.0